docs/ConfigsAndApplications.md

```diff
@@ -147,7 +148,8 @@ minimal application will serve as a general template – the full definition of
             <td><code>String/Array of String</code></td>
             <td>A filename (or array of these) of other config files which are to be included into this application. These names may begin with a <a href="http://docs.fluidproject.org/infusion/development/NodeAPI.html#node-js-module-apis">module-relative path</a>
             such as <code>%kettle</code> or else will be interpreted as 
-            paths relative to this config's location in the filesystem. Each filename listed here will be loaded and resolved as a grade and then merged with the 
+            paths relative to this config's location in the filesystem. The filenames may either end with a <code>.json</code> or a <code>.json5</code> extension representing configuration files in those formats, or the extension may be
+            omitted in which case both of those extensions will be tried as possibilities. Each filename listed here will be loaded and resolved as a grade and then merged with the 
```

Comment:
Is it possible that we would have a situation where we had both a ".json" and a ".json5" file? I'm wondering if we should document the order that we try the filenames in? It would mean that we would need to keep the docs in sync if we ever changed the order that we try the names in (and therefore might not be worth documenting).